test/app.test.js

```diff
@@ -84,13 +84,12 @@ describe("app", function() {
 
           it('should validate the input', function () {
               return tester
-                  .setup.user.state('Facility_Code_Entry', {
-                      creator_opts: {
-                          error: 'The facility code is invalid'
-                      }
+                  .setup.user.state('Facility_Code_Entry')
+                  .input('123') // invalid, not in fixtures
+                  .check.interaction({
+                      state: 'Facility_Code_Entry',
+                      reply: /Sorry, that code is not recognised\./
```

Comment:
This test should pass since this error `Sorry, that code is not recognised` is part of the creator opts that is passed back to the original state, except it never is in the `opts` that is given to the state.